src/main/java/net/minecraftforge/gradle/common/BasePlugin.java

```diff
@@ -368,7 +368,7 @@ public boolean isSatisfiedBy(Dependency element)
         }), null);
         if (fgDep == null) {
             // This shouldn't happen, unless people are doing really wonky stuff
-            throw new StopExecutionException("Missing FG dep in buildscript classpath");
```

Comment:
FG needed to be copied to the forked decompile task's classpath so everything worked. This was to cause early failures if it couldn't be located.